.github/workflows/ci_cron_weekly.yml

```diff
@@ -100,7 +100,7 @@ jobs:
     # of using system libraries and using pytest directly.
 
     runs-on: ubuntu-20.04
-    name: Python 3.9
+    name: Python 3.10
```

Comment:
Why do we put Python version in the name? It makes no sense. 😆

Reply:
It does not control the Python version. Even on `main`, it already picks up Python 3.10 even though we say Python 3.9 in the name: https://github.com/astropy/astropy/actions/runs/4593442983/jobs/8111372319 

🤷